ceph.spec.in

```diff
@@ -1100,6 +1100,16 @@ Group:          System/Monitoring
 This package provides Ceph’s default alerts for Prometheus.
 %endif
 
+%package dashboard-e2e
```

Comment:
could you guard this with an option which is off by default?

Reply:
That would defeat the purpose of the PR - i.e. to enable running the dashboard E2e tests in teuthology.

Reply:
but i think we can always enable it. and more importantly, i don't think the downstream maintainers would be interested in packaging the E2E tests.

Reply:
Hm, why do you think there would be no interest in running the Dashboard E2E tests?

Reply:
say, i am a Ceph user, and i'd like to upgrade my cluster to octopus, why would i want to run Dashboard E2E tests? if not, what's the point of packaging it for nobody?

Reply:
Not sure what you mean by "Ceph user"? There are many people using Ceph, and they have a range of reasons for doing so. I think there is a subset of "Ceph users" who are interested in integration testing of Ceph packages, and below I provided an explanation why I believe packaging the test code is superior to cloning it from a git repo.

If we package it, but disable it from being built, the package cannot be used for testing, or indeed any other purpose.